tests/small/test_frameworks_loader.py

```diff
@@ -1192,7 +1192,7 @@ class _CustomFramework(umake.frameworks.BaseFramework):
 
         def __init__(self):
             super().__init__(name="Custom", description="Custom uninstallable framework",
-                             category=umake.frameworks.MainCategory())
+                             category=umake.frameworks.MainCategory(), force_loading=False)
```

Comment:
hum, force_loading should be False ion the framework by default. I would prefer that you set that on the framework itself and don't touch those tests.